src/services/AbstractPalletsService.ts

```diff
@@ -79,7 +91,7 @@ export abstract class AbstractPalletsService extends AbstractService {
 				metadataFieldType
 			);
 			filtered = pallets.filter(
-				(mod) => (mod[metadataFieldType] as typeof palletMetaType).isSome
+				(mod) => !(mod[metadataFieldType] as typeof palletMetaType).isEmpty
```

Comment:
Whats the motivation behind this change? It's the same?

Reply:
I changed this to `isEmpty` because the outer type of the `constants` field on the `PalletMetadataV14` interface is a `Vec` (`Vec<PalletConstantMetadataV14>`) and not an `Option` like in the other cases that are handled(e.g. `Option<PalletStorageMetadataV14>`).  The property `isSome` doesn't exist for the `Vec` but `isEmpty` exists for both the `Option` and `Vec` types and checks if either of them are empty.

Reply:
Ahh that makes total sense thanks for the clarification